go/utils.go

```diff
@@ -157,3 +157,25 @@ func WaitForTask(taskRef xenapi.TaskRef, delay int) error {
 	}
 	return nil
 }
+
+func WaitForSRReady(srRef xenapi.SRRef) error {
+	token := ""
+	maxTries := 3
+	eventTypes := []string{"PBD"}
+	for i := 0; i < maxTries; i++ {
+		eventBatch, err := xenapi.Event.From(session, eventTypes, token, 10.0)
```

Comment:
Where does this `session` come from?
I understand it should be passed in as a parameter of this function as well.

Reply:
It's a global value. Defined in base_test.go